spring-grpc-core/src/main/java/org/springframework/grpc/client/DefaultGrpcChannelFactory.java

```diff
@@ -71,68 +66,41 @@ public void setCredentialsProvider(ChannelCredentialsProvider credentials) {
 	}
 
 	@Override
-	public ManagedChannelBuilder<?> createChannel(String authority) {
-		return this.createChannel(authority, List.of(), false);
-	}
-
-	@Override
-	public ManagedChannelBuilder<?> createChannel(String authority, List<ClientInterceptor> interceptors,
-			boolean mergeWithGlobalInterceptors) {
-		Assert.notNull(interceptors, () -> "interceptors must not be null");
-		return this.builders.computeIfAbsent(authority, path -> {
-			ManagedChannelBuilder<?> builder = newChannelBuilder(this.targets.getTarget(path),
-					this.credentials.getChannelCredentials(path));
-			this.interceptorsConfigurer.configureInterceptors(builder, interceptors, mergeWithGlobalInterceptors);
```

Comment:
Will have to pass the interceptors configurer to the new builder to let it do its magic (or something like it)